TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php

```diff
@@ -86,6 +94,14 @@ public function publishCommand($collection = null)
                 $this->outputLine('Publishing resources of collection "%s"', array($collection->getName()));
                 $collection->publish();
             }
+
+            if ($this->messageCollector->hasNotification()) {
+                $this->outputLine();
+                $this->outputLine('The system has generate some notifications. You can check and probably fix the integrity of the resource registry by using the resource:clean command.');
```

Comment:
A little typo ("generate*d*"), but maybe in general rephrase like this?

"The resources were published, but a few inconsistencies were detected. You can check and probably fix the integrity of the resource registry by using the resource:clean command."